core/drivers/sub.mk

```diff
@@ -39,5 +39,6 @@ srcs-$(CFG_IMX_RNGB) += imx_rngb.c
 
 subdirs-y += crypto
 subdirs-$(CFG_BNXT_FW) += bnxt
+subdirs-$(CFG_CLK) += clk
```

Comment:
Maybe rename `CFG_CLK` to an explicit `CFG_CLK_DRIVERS` or `CFG_GENERIC_CLK`?
(platforms can implement clocks without `CFG_CLK` (as they currenly do) so I feel the config switch name a bit confusing).

Reply:
Yes, I'll rename that to CFG_GENERIC_CLK.